Dockerfile

```diff
@@ -92,7 +92,10 @@ RUN cd /tmp/test-infra && \
   rm -rf /tmp/test-infra
 
 # Install Agent / test requirements
-RUN pip3 install -r https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/main/requirements.txt && \
+# Remove AWS-related deps as we already install AWS CLI v2
+RUN curl -fsSL https://raw.githubusercontent.com/DataDog/datadog-agent-buildimages/main/requirements.txt | \
+  grep -ivE "boto3|botocore|awscli|urllib3" > requirements-agent.txt && \
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
🤯